src/pages/index.js

```diff
@@ -71,24 +121,105 @@ addCardAddButton.addEventListener("click", () => {
 
 updateProfileButton.addEventListener("click", () => {
   const userData = userInfo.getUserInfo();
-  profileHeadingInput.value = userData.userName;
-  profileDescriptionInput.value = userData.userDescription;
-  editProfileFormValidator.resetValidation(); //editProfileFormValidator might need to change back Not sure what my mistake is//
+  profileHeadingInput.value = userData.name;
+  profileDescriptionInput.value = userData.about;
+  editProfileFormValidator.resetValidation();
   updateProfilePopup.open();
 });
 
+updateAvatarButton.addEventListener("click", () => {
+  avatarPopup.open();
```

Comment:
Please, call `toggleButtonState` for the avatar form to disable the button if the input is empty on opening

Reply:
I'm not sure what to do here.